backends-velox/src/test/scala/org/apache/gluten/execution/TestOperator.scala

```diff
@@ -2078,4 +2078,12 @@ class TestOperator extends VeloxWholeStageTransformerSuite with AdaptiveSparkPla
       checkGlutenOperatorMatch[SortExecTransformer]
     }
   }
+
+  // Open it after fix
+  // test("Test round expression") {
+  //   val df1 = runQueryAndCompare("SELECT round(cast(0.5549999999999999 as double), 2)") { _ => }
+  //   checkLengthAndPlan(df1, 1)
+  //   val df2 = runQueryAndCompare("SELECT round(cast(0.19324999999999998 as double), 2)") { _ => }
+  //   checkLengthAndPlan(df2, 1)
+  // }
```

Comment:
Comparing to commenting the code out, you can use `ignore` scalatest flag, e.g.,

```scala
ignore("Test round expression") {
  ...
}
```